package.json

```diff
@@ -45,7 +45,10 @@
     "onCommand:PowerShell.UnregisterExternalExtension",
     "onCommand:PowerShell.GetPowerShellVersionDetails",
     "onView:PowerShellCommands",
-    "onNotebookEditor:PowerShellNotebookMode"
+    "onNotebook:PowerShellNotebookModeDefault",
+    "onNotebookEditor:PowerShellNotebookModeDefault",
+    "onNotebook:PowerShellNotebookModeOption",
+    "onNotebookEditor:PowerShellNotebookModeOption"
```

Comment:
in a few insider builds we can remove the onNotebookEditor ones as that will be deprecated.